### PR TITLE
SW-4891 Deliverable list redux and mock data

### PR DESCRIPTION
**Pull request by nickgraz**
- Implement deliverables redux feature
- Implement DeliverableService with mock data
- There are a bunch of TODOs for what needs to be updated when the BE is done
- For now just returning the same dataset regardless of whether the request is made from admin or participant side

### CODE REVIEW

**Comment by nickgraz** [comment]
Current dependencies on/for this PR:
* **#2185** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2185?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2184** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2184?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2184?utm_source=stack-comment).

File: src/redux/features/deliverables/deliverablesThunks.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { Dispatch } from 'redux';
+import { RootState } from 'src/redux/rootReducer';
+import { SearchCriteria, SearchSortOrder } from 'src/types/Search';
+import DeliverablesService from 'src/services/DeliverablesService';
+import { setDeliverableListAction } from 'src/redux/features/deliverables/deliverablesSlice';
+
+export const requestDeliverableList = (
```
Comment (karthikbtf) [inline src/redux/features/deliverables/deliverablesThunks.ts:7]: use async thunks so we can leverage the status for ux purposes?

**Review by karthikbtf — APPROVED** [review]
Thanks, this is a good starting point 🙏

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 22, 4:12 PM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2184).

File: src/redux/features/deliverables/deliverablesThunks.ts (hunk 1 above)
Comment (nickgraz) [reply src/redux/features/deliverables/deliverablesThunks.ts:7]: I have another PR coming that converts these

**Pull request merged**